src/components/Login/InvalidUserMessage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { makeStyles, Typography, Box } from '@material-ui/core';
+import { Warning } from '@material-ui/icons';
+
+const useStyles = makeStyles({
+  outlinedBox: {
+    width: '404px',
+    height: '104px',
+    border: '1px solid #AC3434',
+    boxSizing: 'border-box',
+    borderRadius: '10px',
+    paddingLeft: '26px',
+    paddingTop: '17px',
+    marginBottom: '61px',
+  },
+  warningIcon: {
+    width: '22.3px',
+    height: '19px',
+  },
+  invalidUserText: {
+    display: 'inline-block',
+    paddingLeft: '19.7px',
+    paddingBottom: '7px',
+  },
+});
+
+const InvalidUserMessage = () => {
+  const classes = useStyles();
+  return (
+    <Box className={classes.outlinedBox}>
+      <Box>
+        <Warning color="error" className={classes.warningIcon} />
+        <Typography
+          color="error"
+          className={classes.invalidUserText}
+          variant="caption"
+        >
+          Invalid User
+        </Typography>
+      </Box>
+      <Typography color="error" variant="caption">
+        Sorry, you don’t have access to the system yet! Please see administrator
```

Comment:
I am assuming the line break is from the linter?

Reply:
neither of these should hit the [80 char limit which is what prettier enforces](https://github.com/uwblueprint/paramedics-react/blob/master/.prettierrc#L2) - i think it hurts readability so let's just make sure static strings can stay on one line (when short enough like these)

Reply:
yeah this change is being caused by the linter. I tried moving it to one line, but `npm run lint` forces it to be two. Maybe the linter (eslint or prettier) counts the white space indents in the char count?